setup.py

```diff
@@ -254,6 +254,8 @@ def run(self):
 
 extras["serving"] = deps_list("pydantic", "uvicorn", "fastapi", "starlette")
 extras["speech"] = deps_list("soundfile", "torchaudio")
+extras["tf-speech"] = deps_list("soundfile")
+extras["flax-speech"] = deps_list("soundfile")
```

Comment:
Should we change this slightly and have a `"speech"` dependency with just `soundfile` and then a `"torchaudio"` dependency or a `torch-speech` dependency?

Reply:
It's for super specific installs, but maybe not. Wdyt @LysandreJik ?

Reply:
Maybe deprecating `speech` in favor of `torch-speech` so that we have all three covered?